.github/ISSUE_TEMPLATE/release.md

```diff
@@ -4,98 +4,147 @@ about: Instructions and checklist for creating a release.
 title: 'Release X.Y.Z-rc.N'
 labels: kind/release
 assignees: ''
-
 ---
 
-# Release {version}
+# Open Match Release Process
+
+Follow these instructions to create an Open Match release.  The output of the
+release process is new images and new configuration.
+
+## Getting setup
+
+*note: the commands below are pasted from the 0.5 release.  make the necessary
+changes to match your naming & environment.*
+
+The Git flow for pushing a new release is similar to the development process
+but there are some small differences.
+
+**1. Clone your fork of the Open Match repository.**
+
+```shell
+git clone git@github.com:afeddersen/open-match.git
+```
+**2. Move into the new open-match directory.**
```

Comment:
Merge with previous step.

Reply:
If the instructions are correct I would prefer not to go back and forth on style preferences.